README.md

```diff
@@ -22,6 +22,8 @@ of foreman_scap_client.
   Unless set to an alternative file source, URL will be used.
 * 'foreman_repo_src':  Alternative baseurl for The Foreman plugins repository
 * 'foreman_repo_gpg_chk': Enable / disable GPG checks. Directly passed to Yumrepo resource
+* 'package_name': Name of the package that will be installed, defaults to `rubygem-foreman_scap_client`
```

Comment:
The default is platform dependent.

Reply:
Cleaned this up, after a little whupsie - documentation also adjusted to note package name is platform specific